### PR TITLE
Add user notification

### PR DESCRIPTION
**Pull request by WNjihia**
Closes issue #5 
When a data request is added, organization members are notified via email.

### CODE REVIEW

File: ckanext/datarequests/templates/emails/notify_user_body.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Dear {{ user_name }},
+
+A new data request has been added to {{ site_title }}:
+
+Title: {{ datarequest_title }}
+Description:
+{{ datarequest_description }}
+
+You can access it at {{ datarequest_url }} .
```
Comment (DavidLemayian) [inline ckanext/datarequests/templates/emails/notify_user_body.txt]: I've added the URL here, please add it to function.

File: ckanext/datarequests/controllers/ui_controller.py [hunk 2]
```diff
@@ -174,7 +176,25 @@ def _process_post(self, action, context):
 
             try:
                 result = tk.get_action(action)(context, data_dict)
-                base.redirect(helpers.url_for(controller='ckanext.datarequests.controllers.ui_controller:DataRequestsUI', 
+                if result['id']:
+                    users = result['organization']['users']
```
Comment (DavidLemayian) [inline ckanext/datarequests/controllers/ui_controller.py]: Just a question here: is this limited to the organization to that the DataRequest has been submitted to?

File: ckanext/datarequests/templates/emails/notify_user_body.txt (hunk 1 above)
Comment (WNjihia) [reply ckanext/datarequests/templates/emails/notify_user_body.txt]: Okay, I will

File: ckanext/datarequests/controllers/ui_controller.py (hunk 2 above)
Comment (WNjihia) [reply ckanext/datarequests/controllers/ui_controller.py]: Yes, when a new data request is added, the members of the organization are notified

File: ckanext/datarequests/templates/emails/notify_user_body.txt (hunk 1 above)
Comment (WNjihia) [reply ckanext/datarequests/templates/emails/notify_user_body.txt]: @DavidLemayian I have added the function to add the datarequest_url

**Pull request closed without merging**